docker-compose.yml

```diff
@@ -52,13 +52,48 @@ services:
     environment:
       NUM_THREADS: 8 # default value for NUM_THREADS
       CCACHE_DIR: "/velox/.ccache"
-      EXTRA_CMAKE_FLAGS: -DVELOX_ENABLE_PARQUET=ON 
+      EXTRA_CMAKE_FLAGS: -DVELOX_ENABLE_PARQUET=ON
                          -DVELOX_ENABLE_S3=ON
     volumes:
       - .:/velox:delegated
     working_dir: /velox
     command: /velox/scripts/docker-command.sh
 
+  adapters-cuda:
```

Comment:
This is copy of `adapters-cpp`, but with GPU support configured. See the comment below.